m4_sessions/1708-inning/group_five.md

```diff
@@ -1,9 +1,21 @@
 # Group Five
 
-### Session Title:
+### Session Title: The Robots are Coming: The future of AI
 
 ### Session Outcomes: 
+- What does it mean to me as a programmer
+- Want people to consider how AI benefits or threatens their lives as devs
+- What AI is currently available, near future, and distant future
+- Understanding of what AI is (definitions and relevant vocabulary)
 
 ### Session Outline:
+- 10 to 15 min Intro (Icebreaker)
+- 20 min content video
+- Discuss the video
+- Break out in to small groups and discuss 
+  - AI in your lives and how it could affect you our those you know
+  - What kind of AI have you seen and what might you like to create or see create
+- Closing discussion on what the groups talked about and final thoughts from mod 4
```

Comment:
What types of final thoughts from M4?  Your own opinions?  What you are considering in the job search?